ddht/v5_1/rpc_handlers.py

```diff
@@ -196,6 +196,27 @@ async def do_call(self, params: NodeID) -> None:
         return None
 
 
+class LookupENRHandler(RPCHandler[Tuple[NodeID, Optional[int]], GetENRResponse]):
+    def __init__(self, network: NetworkAPI) -> None:
+        self._network = network
+
+    def extract_params(self, request: RPCRequest) -> Tuple[NodeID, Optional[int]]:
+        raw_params = extract_params(request)
+        raw_destination = raw_params[0]
+        node_id, _ = validate_and_extract_destination(raw_destination)
```

Comment:
Odd to me that we're discarding the `endpoint` part of the return value.  If this is a node we haven't seen yet (and thus it isn't in our ENR database) this will make the lookup go a log slower because it'll cause it to have to fallback to a `recursive_find_nodes` approach.